fakeApi.js

```diff
@@ -33,6 +33,16 @@
         license:'GPL'
     }];
 
+    var userData =[
```

Comment:
Only one single user can be logged in the system, so, make the user info an object instead of an Array.